src/store/ws/zone.store.ts

```diff
@@ -14,6 +14,11 @@ class ZoneStore {
   samples: string[] = []
   zone: any[] = []
 
+  private _fetchingGenes: boolean = false
```

Comment:
Please, wait for zone store refactoring, it will provide async stores for zones

Reply:
Done